utils/channel_utils.jsx

```diff
@@ -47,3 +49,12 @@ export function findNextUnreadChannelId(curChannelId, allChannelIds, unreadChann
 
     return -1;
 }
+
+export function makeGetChannelUrlById(state) {
```

Comment:
Maybe safe to remove since it's not being used.

Reply:
Totally forgot to remove it, will do soon